--- origsrc/xcftools-1.0.4/Makefile.in	2006-02-22 14:00:19.000000000 -0600
+++ src/xcftools-1.0.4/Makefile.in	2007-05-16 11:58:08.758621200 -0500
@@ -30,8 +30,8 @@
 program_transform_name	= @program_transform_name@
 
 CFLAGS			= @CFLAGS@
-LDFLAGS			= @LDFLAGS@ @LIBICONV@
-LIBS			= @LIBS@
+LDFLAGS			= @LDFLAGS@
+LIBS			= @LIBS@ @LTLIBINTL@ @LTLIBICONV@
 
 INSTALL			= @INSTALL@
 INSTALL_PROGRAM		= @INSTALL_PROGRAM@
@@ -70,21 +70,21 @@
 all: $(BINARIES) $(SCRIPTS) $(MANPAGES) po/stamp manpo/all
 
 xcfinfo$e: xcfinfo.$o $(FILEIO).$o enums.$o xcf-general.$o utils.$o nlsini.$o
-	$(CC) $(LDFLAGS) $(LIBS) $^ -o $@
+	$(CC) $(LDFLAGS) -o $@ $^ $(LIBS)
 
 xcf2pnm$e: xcf2pnm.$o $(FILEIO).$o enums.$o xcf-general.$o utils.$o nlsini.$o \
 	   pixels.$o flatten.$o flatspec.$o scaletab.$o table.$o
-	$(CC) $(LDFLAGS) $(LIBS) $^ -o $@
+	$(CC) $(LDFLAGS) -o $@ $^ $(LIBS)
 
 xcf2png$e: xcf2png.$o $(FILEIO).$o enums.$o xcf-general.$o utils.$o nlsini.$o \
 	   pixels.$o flatten.$o flatspec.$o scaletab.$o table.$o palette.$o
-	$(CC) $(LDFLAGS) $(LIBS) -lpng $^ -o $@
+	$(CC) $(LDFLAGS) -o $@ $^ $(LIBS) -lpng
 
 xcfview: xcfview.in Makefile
 	sed '1s,/usr/bin/perl,@PERL@,' < $< > $@
 
 testscale$e: testscale.$o
-	$(CC) $(LDFLAGS) $(LIBS) $^ -o $@
+	$(CC) $(LDFLAGS) -o $@ $^ $(LIBS)
 
 install: all
 	for p in $(BINARIES) ; do \
